src/pages/Overview/OverviewPage.tsx

```diff
@@ -299,6 +345,47 @@ class OverviewPage extends React.Component<OverviewProps, State> {
       </>
     );
   }
+
+  renderStatuses(ns: NamespaceInfo): JSX.Element {
```

Comment:
What about having an smaller component with those Statuses? Is it too much fragmentation? I see this component is growing to 400L. I think is a smell that things should be moved. wdyt?